src/js/component/home.jsx

```diff
@@ -7,14 +7,15 @@ import "../../styles/index.scss";
 
 //create your first component
 const Home = () => {
-	let BASENAME = "https://208b-77-230-204-78.ngrok.io";
+	let BASENAME = "https://8400-77-230-204-78.ngrok.io";
```

Comment:
I don't think we should be fixing the location of the host we're going to use. This code would be more useful if this url is received from the command line.